pom.xml

```diff
@@ -42,7 +48,7 @@
                     <dependency>
                         <groupId>com.puppycrawl.tools</groupId>
                         <artifactId>checkstyle</artifactId>
-                        <version>8.29</version>
+                        <version>9.0.1</version>
```

Comment:
Seems unrelated to this PR. Please do a separate Pr for it if you want to update it.

Reply:
OK